Jellyfin.Plugin.OpenSubtitles/OpenSubtitleDownloader.cs

```diff
@@ -31,6 +31,7 @@ public class OpenSubtitleDownloader : ISubtitleProvider
     private DateTime? _limitReset;
     private DateTime? _lastRatelimitLog;
     private List<string>? _languages;
+    private List<int> _badSubtitleIds = new ();
```

Comment:
This isn't thread-safe but can be accessed from multiple threads

Reply:
Would a `ConcurrentBag` be ok?